### DISCUSSION TITLE
increase the number of entries per page [May 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2233] [Discord Threads]
> 🙋‍♂️ **Mr.Raccoon**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### you can configure using env var `DB_QUERY_LIMIT_DEFAULT`   
#### See https://docs.nocodb.com/getting-started/installation#environment-variables  
#  
> 🙋‍♂️ **wkw**  
#### however,  if your target is above `DB_QUERY_LIMIT_MAX`, then you may need to increase it as well  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978691684349128785)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
